packages/list-reusable-blocks/src/style.scss

```diff
@@ -3,7 +3,11 @@
 
 .list-reusable-blocks__container {
 	display: inline-flex;
-	padding: 9px 0 4px; // To match the H1
 	align-items: center;
-	vertical-align: top;
+	position: relative;
+	top: -3px;
+
+	.components-button {
```

Comment:
Perhaps there should be a comment here noting that this is a temporary workaround until the core styles get updated?

Reply:
Updated 👍